jaypeasee/national-parkfinder
# Hotfix - Saved List Bug

## Is this a fix or a feature?
Fix

## What does it fix?
Fix a toggling bug in the bucket list and visited list views/cards

## Where should the reviewer start?
Review ParkContainer and SavedCard

## How should this be tested?
Run the page, add a park to the bucket list, go to the bucket list, click to add it to visited, go to visited and check to make sure the [Remove from Bucket List] button is no longer there.

@saraho1123
@jaypeasee
@baileydunning
